### PR TITLE
feat-mobile: account toggle visibility & delete

### PR DESCRIPTION
**Pull request by begonaalvarezd**
## Summary

Depends on https://github.com/iotaledger/firefly/pull/5687, nested PR

...

## Changelog

```
feat-mobile: account toggle visibility & delete
```

## Relevant Issues

Close https://github.com/iotaledger/firefly/issues/4970

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte [hunk 1]
```diff
@@ -17,4 +40,25 @@
...
+        <Button
+            outline
+            disabled={!$selectedAccount.hidden && $nonHiddenActiveAccounts.length <= 1}
+            onClick={() => ($selectedAccount.hidden ? handleShowAccountClick() : handleHideAccountClick())}
+            icon={$selectedAccount.hidden ? Icon.View : Icon.Hide}
+        >
+            {localize($selectedAccount.hidden ? 'actions.showAccount' : 'actions.hideAccount')}
+        </Button>
+    {/if}
+    {#if features?.dashboard?.accountActions?.toggleVisibility?.enabled}
```
Comment (paul-boegelsack) [inline packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte:53]: This should be `{#if features?.dashboard?.accountActions?.delete?.enabled}` right?

File: packages/shared/components/Button.svelte [hunk 2]
```diff
@@ -77,7 +77,7 @@
             {#if icon}
                 <Icon
                     {icon}
-                    classes="text-{iconColor}"
+                    classes="text-{iconColor} mx-2"
```
Comment (paul-boegelsack) [inline packages/shared/components/Button.svelte:80]: This is okay for desktop?

File: packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte [hunk 3]
```diff
@@ -17,4 +40,25 @@
...
+        <HR />
+        <Button
+            variant={ButtonVariant.Warning}
+            icon={Icon.Delete}
+            disabled={!$selectedAccount.hidden && $nonHiddenActiveAccounts.length <= 1}
```
Comment (paul-boegelsack) [inline packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte:58]: The message isn't right?
image

The delete button for the same account is hidden on desktop:
image

**Review by paul-boegelsack — CHANGES_REQUESTED** [review]
Looks great! 🔥  I just noticed some differences between desktop.

**Review by paul-boegelsack — APPROVED** [review]
lgtm 🔥 🚀

File: packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte (hunk 1 above)
Comment (begonaalvarezd) [reply packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte]: right

File: packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte (hunk 3 above)
Comment (begonaalvarezd) [reply packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte:62]: The message isnt right, opened discussion with the team, the issue is also found on desktop

**Pull request merged**